extensions/agroal/deployment/src/main/java/io/quarkus/agroal/deployment/AgroalProcessor.java

```diff
@@ -92,6 +93,17 @@ void build(
         for (AggregatedDataSourceBuildTimeConfigBuildItem aggregatedDataSourceBuildTimeConfig : aggregatedDataSourceBuildTimeConfigs) {
             validateBuildTimeConfig(aggregatedDataSourceBuildTimeConfig);
 
+            if (aggregatedDataSourceBuildTimeConfig.getJdbcConfig().tracing) {
+                if (!QuarkusClassLoader.isClassPresentAtRuntime(DataSources.TRACING_DRIVER_CLASSNAME)) {
```

Comment:
This will be checked by `validateBuildTimeConfig` right?

Reply:
yes